appservice/src/index.ts

```diff
@@ -3,4 +3,10 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+export * from './AzureAccountWrapper';
+export * from './errors';
 export * from './SiteWrapper';
+export * from './utils/ui';
+export * from './utils/random';
+export * from './WebsiteCreatorBase';
```

Comment:
It seems like you intend for each extension to extend WebsiteCreatorBase? As I said above, I would prefer to keep the api surface low and this would not do that

In addition, Chris mentioned he might have a 'Create App Service' entry point from the docker extension. It doesn't make sense for him to _also_ implement 'WebsiteCreator' when we could take care of it for him in this package